### DISCUSSION TITLE
请问自行编译在Firefox上运行需要对源码文件进行哪些修改

### DISCUSSION THREAD

**Discussion by xxnuo** [chunibyocola/sc-translator-crx#113] [Q&A]
请问自行编译在Firefox上运行需要对源码文件进行哪些修改呢？

因为我想修改一下样式自行使用，编译后在Firefox调试工具内会显示`chrome.action`undefined

Firefox用的manifest.json我在xpi文件中找到了，但是background.js内还包含某些不同，请问是修改了哪些代码呢？😋

**Comment by chunibyocola** [comment]
这是需要进行比较复杂的修改的，因为这两者标准不一样，我自己也是两套代码去改，每次发布新版本都浪费极多时间，我不建议自己这样改，使用油猴脚本去更改样式更好。

**Reply by xxnuo** [reply]
好的，我试试用油猴脚本替换样式🤣
